js/functions.js

```diff
@@ -0,0 +1,33 @@
+const checkString = (testString, maxLength) => testString.length <= maxLength;
+
+const isPalindrome = (testString) => {
+  testString = testString.toLowerCase().replaceAll(' ', '');
+
+  const LAST_SYMBOL = testString.length - 1;
+  for (let i = 0; i < testString.length / 2; i++) {
+    return testString.at(i) === testString.at(LAST_SYMBOL - i);
+  }
+};
+
+const findsNumbers = (string) => {
+  if (typeof string === 'number') {
+    return string;
+  }
+  let result = '';
+  for (let i = 0; i < string.length; i++) {
+    if (!Number.isNaN(parseInt(string.at(i), 10))) {
+      result += string.at(i);
+    }
+  }
```

Comment:
Лучше форматировать код так, отделять циклы и if одной пустой строкой
```suggestion
  if (typeof string === 'number') {
    return string;
  }
  
  let result = '';
  
  for (let i = 0; i < string.length; i++) {
    if (!Number.isNaN(parseInt(string.at(i), 10))) {
      result += string.at(i);
    }
  }
```